### ISSUE TITLE
Import functions from other files in pre-request script

### ISSUE THREAD

**Issue by detarkende** [rangav/thunder-client-support#920]
I tried importing my util function in a custom pre-request filter 2 ways: commonjs and ES modules but both gave me an error.
How could I share functions between files?

1.
```js
import { sendRequest } from './util/requests.js';
```
`Error in Custom Filter: auth - 'import' and 'export' may appear only with 'sourceType: module'`

2.
```js
const { sendRequest } = require('./util/requests.js');
```
`Error in Custom Filter: auth - Cannot destructure property 'sendRequest' of 'tc.getModule(...)' as it is undefined.`

**Comment by rangav** [comment]
@detarkende currently not possible to read code from a file. You need to copy that code to main `js` file.

Will research and see what is the best way to implement this.

**Comment by detarkende** [comment]
Got it. Thanks for the quick response. I guess I'll use a bundler for the time being then.

**Comment by rangav** [comment]
This feature is implemented and published to the marketplace, please update to `v2.16.0`

See all features in this update
https://github.com/rangav/thunder-client-support/releases/tag/v2.16.0

Please test and let me know your feedback.
